<launch>
  <node name="ob1_odometry_publisher" pkg="navigation" type="odom_publisher.py" output="screen"></node>
</launch>